catalyst_voices/apps/voices/lib/routes/routing/router_refresh_stream.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+
+final class RouterRefreshStream extends ChangeNotifier {
```

Comment:
Is this class needed?

Reply:
Yeah, not really. Leftover from trying to implement `refreshListenable`